sable_network/src/history/local_service.rs

```diff
@@ -20,12 +22,12 @@ fn target_id_for_entry(for_user: UserId, entry: &HistoryLogEntry) -> Option<Targ
 }
 
 /// Implementation of [`HistoryService`] backed by [`NetworkNode`]
-pub struct LocalHistoryService<'a> {
-    node: &'a NetworkNode,
+pub struct LocalHistoryService<'a, NetworkPolicy: policy::PolicyService> {
```

Comment:
I'm starting to really question whether making Node generic over its policy was a good idea. Not much to do about it in this PR but it wants thought more generally.